src/Core/ImageFunction/ResampleSegmentedImage/Code.cxx

```diff
@@ -100,13 +97,9 @@ main(int argc, char * argv[])
   gaussianInterpolator->SetAlpha(3.0);
   resizeFilter->SetInterpolator(gaussianInterpolator);
 
-  using WriterType = itk::ImageFileWriter<ImageType>;
-  WriterType::Pointer writer = WriterType::New();
-  writer->SetInput(resizeFilter->GetOutput());
-  writer->SetFileName(outputImageFileLabelImageInterpolator);
   try
   {
-    writer->Update();
+    itk::WriteImage(resizeFilter->GetOutput(), outputImageFileLabelImageInterpolator);
```

Comment:
```suggestion
    itk::WriteImage(resizeFilter->GetOutput(), outputImageFileLabelImageInterpolator, true); // compress 
```
When writing label images, compressing them is quite desirable.

Reply:
Done